__tests__/articles.test.js

```diff
@@ -0,0 +1,87 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    describe("Status 200", () => {
```

Comment:
This test layout is very well organised with the use of nested describe blocks 👍